assets/css/global.css

```diff
@@ -37,7 +37,7 @@ h2 {
 }
 
 h3 {
-  @apply text-xl;
+  @apply text-base lg:text-xl;
```

Comment:
text-base class is redundant here, its already applied in the body tag selector.
just keep lg:text-xl